release-squad.md

```diff
@@ -0,0 +1,88 @@
+
+So to be clear: **It’s OKAY for the CfT to be edited/updated during the release cycle, or for follow-up CfTs to be published afterwards.** The post's change log is used to track updates, and major updates may even warrant detailed separate posts in Make/Core. If you’ve followed past releases, you'll notice there are always adjustments made because we're all human, and things fall through the cracks. But most features are typically settled by RC1.
+
+### Post Format
+The format of the CfT was adapted from the [WordPress 5.9 CfT](https://make.wordpress.org/test/2021/11/30/help-test-wordpress-5-9-features/), where some of the post is boilerplate, but the majority of content is specific to new release features.
+
+A great way to start the CfT is to make a copy of the document used by the previous Test Team release squad, which is often co-authored in Google Docs.
+
+### Source Information
+Info related to features introduced in the release comes from a variety of sources, including:  
+
+- “Roadmap” posts, e.g. [https://make.wordpress.org/core/2022/06/04/roadmap-to-6-1/](https://make.wordpress.org/core/2022/06/04/roadmap-to-6-1/).
+- GitHub project boards, e.g. [https://github.com/orgs/WordPress/projects/63](https://github.com/orgs/WordPress/projects/63).
+- Trac release queries, e.g. [enhancements and features for 6.2](https://core.trac.wordpress.org/query?status=accepted&status=assigned&status=new&status=reopened&status=reviewing&type=enhancement&milestone=6.2&or&status=accepted&status=assigned&status=new&status=reopened&status=reviewing&type=feature+request&milestone=6.2&col=id&col=summary&col=status&col=owner&col=type&col=priority&col=milestone&order=priority).
+- “Source of Truth” documents (which typically come _after_ beta, but for which drafts are shared quite early).
+- Release walkthroughs (which are inconsistently timed, and usually *after* beta 😔).
+- Guidance from the release coordinators or previous Test release leads.
+
+That’s a LOT of info, and it can be helpful to create a CfT planning document to sort out a curated list of items to include. You might also use a are color-coding system to indicate whether the source info is good (🟢), just started (🟡), or needs a lot of work (🔴).
+
+> 💡 For shared Google Docs, make sure to keep the link private among the release team. And use the "document outline" feature to quickly jump between sections.
+
+> **A note on the copy:** Please don’t use the provided text as-is; **rewrite or rephrase instead**. These are original raw reference notes, and won’t make sense in the context of the CfT post. Also keep a watch for inaccuracies in the copy, since it may not be 100% correct (it’s a LOT of info that has to be parsed out of GitHub, Trac, posts, etc 😅).
```

Comment:
```suggestion
<div class="callout callout-info">
	**A note on the copy:** Please don’t use the provided text as-is; **rewrite or rephrase instead**. These are original raw reference notes, and won’t make sense in the context of the CfT post. Also keep a watch for inaccuracies in the copy, since it may not be 100% correct (it’s a LOT of info that has to be parsed out of GitHub, Trac, posts, etc 😅).
</div>
```
Same as previous note above.